### PR TITLE
Add TiCDC TSO acquisition in sync-diff-inspector doc

### LINKED ISSUE #11217: support TiCDC TSO in sync-diff-inspector docs
## Change Request

**This repository is ONLY used to solve problems related to DOCS-CN.
For other issues (TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

https://docs.pingcap.com/zh/tidb/stable/upstream-downstream-diff

TiDB主从集群校验，只描述了TiDB Binlog关于上下游 TSO 获取的描述，没有 TiCDC 相关描述。

2. Describe your suggestion or addition.

在 TiDB主从集群校验 文档中，提供 TiCDC 关于上下游 TSO 获取的描述。

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by eastfisher**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Fix #11217

Add TiCDC TSO acquisition in sync-diff-inspector doc

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by eastfisher** [comment]
Why is this PR merged first? https://github.com/pingcap/docs-cn/pull/11273

**Comment by eastfisher** [comment]
@TomShawn

**Comment by TomShawn** [comment]
> 

Thank you very much for creating this PR and contributing to TiDB documentation. We greatly appreciate your work. The TiCDC syncpoint is an experimental feature. It should have been properly documented as an experimental feature, but to our regret, it is not documented at all. Users cannot find the keyword "syncpoint".

As the TiCDC team plans to make it GA in the upcoming v6.3, they created https://github.com/pingcap/docs-cn/pull/11273 to fully introduce the concept of syncpoint. I think this PR should have been reviewed by the TiCDC team @asddongmen instead of @3pointer. Anyway, I think this PR is still a very good complement to https://github.com/pingcap/docs-cn/pull/11273. @asddongmen Could you please take a look?

**Comment by asddongmen** [comment]
I'm very sorry for not noticing your PR before. The TiCDC team has improved the `syncpoint` function and modified some configuration of it. After the work was done, I made changes to this part of the document. Now that the configuration and description of the `syncpoint` function are different from before, I have modified the content of your PR after review.  Thanks a lot for your contribution!

**Review by leoppro — APPROVED** [review]
LGTM

File: sync-diff-inspector/upstream-downstream-diff.md [hunk 1]
```diff
@@ -26,8 +26,7 @@ sync-point-retention = "1h"
 
 ## 获取 ts-map
 
-在下游 TiDB 中执行以下 SQL 语句：
-
+在下游 TiDB 中执行以下 SQL 语句，可以从结果中可以获取上游TSO（primary_ts）和下游TSO（secondary_ts）信息。
```
Comment (shichun-0415) [inline sync-diff-inspector/upstream-downstream-diff.md:29]:
```suggestion
在下游 TiDB 中执行以下 SQL 语句，从结果中可以获取上游 TSO (primary_ts) 和下游 TSO (secondary_ts) 信息。
```

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**